roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -464,6 +464,8 @@ apiserver_loadbalancer_domain_name: "lb-apiserver.kubernetes.local"
 kube_apiserver_global_endpoint: |-
   {% if loadbalancer_apiserver is defined -%}
       https://{{ apiserver_loadbalancer_domain_name }}:{{ loadbalancer_apiserver.port|default(kube_apiserver_port) }}
+  {%- elif use_localhost_as_kubeapi_loadbalancer is defined -%}
```

Comment:
This shoud be `use_localhost_as_kubeapi_loadbalancer|default(False)`

Reply:
will i need to change the code?

Reply:
If you can create a PR that would be nice yes

Reply:
#7310 - created